saleor/webhook/tests/subscription_webhooks/test_webhook.py

```diff
@@ -0,0 +1,90 @@
+import dataclasses
+import json
+from unittest import mock
+
+from ....core.models import EventDelivery
+from ...event_types import WebhookEventAsyncType, WebhookEventSyncType
+from ...models import Webhook
+from ...transport.asynchronous import trigger_webhooks_async
+from ...transport.synchronous import trigger_webhook_sync
+from .payloads import generate_payment_payload
+
+TEST_ID = "test_id"
+
+
+@dataclasses.dataclass
+class FakeDelivery:
+    id = TEST_ID
```

Comment:
This is probably an old leftover and can be dropped? I don't see it being referenced anywhere in Saleor.